mask.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "mask",
     "summary": "Apply a raster mask",
-    "description": "Applies a mask to a raster data cube. To apply a vector mask use ``mask_polygon()``.\n\nA mask is a raster data cube for which parallel pixels among `data` and `mask` are compared and those pixels in `data` are replaced which are non-zero (for numbers) or `true` (for boolean values). The pixel values are replaced with the value specified for `replacement`, which defaults to `null` (no data).",
+    "description": "Applies a mask to a raster data cube. To apply a vector mask use ``mask_polygon()``.\n\nA mask is a raster data cube for which corresponding pixels among `data` and `mask` are compared and those pixels in `data` are replaced which are non-zero (for numbers) or `true` (for boolean values). The pixel values are replaced with the value specified for `replacement`, which defaults to `null` (no data).\n\nThe data cubes have to be compatible so that each dimension in mask must also be available in the raster data cube with the same name, type, reference system, resolution and labels. Dimensions can be missing in the mask with the result that the mask has to be applied for each label of the missing dimension in the data cube. The process fails if there's a incompatibility found between the raster data cube and the mask.",
```

Comment:
> The process fails if ...

do we have a particular error code for dimension mismatches?

Reply:
I thought there's potentially too many different things to consider and inform a user about so that a single error code / definition would be to general and not very helpful.

Reply:
ok fair eneough